compiler/tests/language_features/branching_error.jm

```diff
@@ -0,0 +1,22 @@
+?#even\nodd\neven\nodd\n
+
+i = 0
+while i < 4
+   if isEven(i)
+      puts("even")
+   else 
+      puts("odd")
+   end
+   i = i + 1
+end
+
+def isEven(n:Int)
+   #lack of inital value causing problems
```

Comment:
Just mention the test in the issue. There's not point tailoring tests to what currently fails.